R/modules.R

```diff
@@ -303,7 +303,6 @@ module <- function(label = "module",
     datanames <- filters
     msg <-
       "The `filters` argument is deprecated and will be removed in the next release. Please use `datanames` instead."
-    logger::log_warn(msg)
```

Comment:
You can move the literal message string directly to `warning` now.

Reply:
right, the warning is right below :) (there were separate `log_warn()` and `warning()`)